server/modules/allocations.js

```diff
@@ -39,29 +43,26 @@ export const findPartnerById = partnerId => new Promise((resolve, reject) => {
     resolve(JSON.parse(result).values.filter(partner => partner.id === partnerId)[0]);
   });
 });
+/**
+ *@desc Fetches placements from allocations, based on the status specified
+ *
+ * @param {string} status - The status of placements to fetch
+ *
+ * @returns {axios} - Promise of the returned placements from allocations
```

Comment:
In this piece of code
```
* @returns {axios} - Promise of the returned placements from allocations
```
I don't think `axios` is a valid type. `Promise` should be suitable here, please update the return type.

Reply:
Thanks for that. Correction made